readme.md

```diff
@@ -228,6 +228,7 @@ All about development on EOSIO platform.
 - [EOS Factory](http://eosfactory.io/) - Python-based EOSIO smart-contract development & unit testing framework.
   - [tokenika/eosfactory](https://github.com/tokenika/eosfactory) - Official GitHub.
 - [SwiftyEOS](https://github.com/ProChain/SwiftyEOS) - Framework for interacting with EOS, written in Swift.
+- [infeos](https://github.com/infiniteXLabs/infeos) - JavaScript based framework for development, building, deploying and unit testing EOSIO dApps
```

Comment:
If you try `awesome-lint` it fails with:
```
  ✖  231:52  List item description must end with proper punctuation                                   remark-lint:awesome/list-item
```
If you could add dot `.` in the end of the sentence, thanks

Reply:
Yeah, sure